src/main/steps/children/find-family-court/FindFamilyCourtPostController.ts

```diff
@@ -0,0 +1,46 @@
+import autobind from 'autobind-decorator';
+import { Response } from 'express';
+
+import { YesOrNo } from '../../../app/case/definition';
+import { AppRequest } from '../../../app/controller/AppRequest';
+import { AnyObject, PostController } from '../../../app/controller/PostController';
+import { Form } from '../../../app/form/Form';
+import { getCourtEmailId } from '../../task-list/utils';
+
+@autobind
+export default class FindFamilyCourtPostController extends PostController<AnyObject> {
+  public async post(req: AppRequest<AnyObject>, res: Response): Promise<void> {
+    const fields = typeof this.fields === 'function' ? this.fields(req.session.userCase) : this.fields;
+    const form = new Form(fields);
+    const { saveAndSignOut, saveBeforeSessionTimeout, _csrf, ...formData } = form.getParsedBody(req.body);
+
+    req.session.errors = form.getErrors(formData);
+
+    Object.assign(req.session.userCase, formData);
+
+    const findFamilyCourt = req.session.userCase.findFamilyCourt;
+    const familyCourtName =
+      findFamilyCourt === YesOrNo.YES
+        ? req.session.userCase.placementOrders![0].placementOrderCourt
+        : req.session.userCase.familyCourtName;
+    const familyCourtEmailId = getCourtEmailId(familyCourtName as string);
+
+    console.log('FindFamilyCourtPostController 24: ' + findFamilyCourt);
+    console.log('FindFamilyCourtPostController 25: ' + familyCourtName);
+    console.log('FindFamilyCourtPostController 26: ' + familyCourtEmailId);
+
+    this.filterErrorsForSaveAsDraft(req);
+
+    req.session.userCase = await this.save(
+      req,
+      {
+        findFamilyCourt,
+        familyCourtName,
+        familyCourtEmailId,
+      },
+      this.getEventName(req)
+    );
+
+    this.redirect(req, res);
```

Comment:
Also need to handle the redirect to specific URI
Please take a look at https://github.com/hmcts/adoption-web/blob/032e5500eb54214ae01d6dbdf1e837ce7187b1a8/src/main/steps/children/adoption-agency/AdoptionAgencyPostController.ts#L34